content/references/http-websocket-apis/commandline-usage.md

```diff
@@ -70,7 +70,7 @@ Daemon mode is the default mode of operation for `rippled`. In addition to the [
 
 The following option has been removed: `--validateShards`. [Removed in: rippled 1.7.0][]
 
-## Stand-Alone Mode Options
+### Stand-Alone Mode Options
```

Comment:
```## Daemon Mode Options```
has two ##. Should Stand-Alone Mode Options match that?

Reply:
With 4e625a2, updated so both are at the `##` level